src/Rings/localization_interface.jl

```diff
@@ -176,13 +177,22 @@ function parent(f::AbsLocalizedRingElem)
 end
 
 ### default functionality for printing
-function Base.show(io::IO, f::AbsLocalizedRingElem)
-  if needs_parentheses(f)
-    print(io, "($(numerator(f)))//($(denominator(f)))")
-  else 
-    print(io, "$(numerator(f))//$(denominator(f))")
-  end
-end
+#function Base.show(io::IO, f::AbsLocalizedRingElem)
```

Comment:
I found these in the Ring interface documentation. But when leaving this code, the compiler complains about double definition of the methods. Is this all encapsulated in the macro `@enable_all_show_via_expressify`? If yes, the documentation should be updated.

Reply:
The purpose of the (currently undocumented) macro `@enable_all_show_via_expressify` is precisely to generate the `Base.show` methods. So either use that macro (recommend in general) or insert the methods manually (can be useful if you need extra code in them), but don't use both.

Reply:
Ok, that makes sense. In this case, I will just remove the extra code, then.